docs/client_reference.rst

```diff
@@ -88,14 +88,14 @@ The client session supports the context manager protocol for self closing.
       that generation. Note that ``Content-Length`` autogeneration can't
       be skipped.
 
-      Iterable of :class:`str` or :class:`~aiohttp.istr` (optional)
+      Iterable of :class:`str` or :class:`~multidict.istr` (optional)
 
    :param aiohttp.BasicAuth auth: an object that represents HTTP Basic
                                   Authorization (optional)
 
    :param version: supported HTTP version, ``HTTP 1.1`` by default.
 
-   :param cookie_jar: Cookie Jar, :class:`AbstractCookieJar` instance.
+   :param cookie_jar: Cookie Jar, :class:`aiohttp.abc.AbstractCookieJar` instance.
```

Comment:
Let's maybe prefix this with a tilde?
```suggestion
   :param cookie_jar: Cookie Jar, :class:`~aiohttp.abc.AbstractCookieJar` instance.
```

Reply:
maybe, original without tilde

Reply:
The original was rendered as a class name w/o a prefix